packaging/README.branch.md

```diff
@@ -0,0 +1,23 @@
+# Branch maintaining
+
+## The `master` branch
+
+The `master` branch contains features and bug fixes since last tagged release.
+
+## The `stable_v<x.y.z>` branch
+
+This branch contains bug fixes since `v<x.y.z>` release.
+For example, the `stable_v0.0.8` will contains backported bug fix patches
+since `v0.0.8` release.
+
+This branch will be used for [Fedora nmstate copr repos][1] for stable build.
```

Comment:
Stable releases could go directly into Fedora, not sure why we need COPR for this.
Also regarding COPR, the repos should be consolidated. IMHO it does not make sense to distinguish between Fedora and EL8, since both use the same SPEC file.